testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/DeployAllDomainTestCase.java

```diff
@@ -447,4 +448,225 @@ public void testLegacyEnableSingleAppDeployment() throws CommandLineException, I
         checkExist(result, cliTestApp2War.getName(), ADDED, ctx);
         checkExist(result, cliTestAppEar.getName(), ENABLED, ctx);
     }
+
+    /**
+     * Disabling all deployed application deployments between all server groups.
```

Comment:
"between all" ==> "present in all server groups"

Reply:
Done